app/controllers/api/v1/courses_controller.rb

```diff
@@ -1,4 +1,9 @@
 class Api::V1::CoursesController < ApplicationController
+  def index
+    @tests = ['hello', 'hi', 'how are you?']
+    render json: @tests
+  end
```

Comment:
- Use a descriptive variable name, example: courses instead of tests
- Please remove the array of already defined values, you can use a `.all` to get all available courses

Reply:
Thanks for your review! I did as you said. Also, I added 'gem faker' as a temporary seed. You can review it now, please